packages/x-charts/src/ChartsTooltip/ChartsTooltipTable.ts

```diff
@@ -41,19 +41,17 @@ export const ChartsTooltipCell = styled('td', {
 })(({ theme }) => ({
   verticalAlign: 'middle',
   color: (theme.vars || theme).palette.text.secondary,
-
   [`&.${chartsTooltipClasses.labelCell}`]: {
     paddingLeft: theme.spacing(1),
   },
   [`&.${chartsTooltipClasses.valueCell}`]: {
     paddingLeft: theme.spacing(4),
     color: (theme.vars || theme).palette.text.primary,
   },
-
-  'td:first-of-type&': {
+  '& td:first-of-type&': {
     paddingLeft: theme.spacing(2),
   },
-  'td:last-of-type&': {
+  '& td:last-of-type&': {
```

Comment:
```suggestion
  '&td:last-of-type&': {
```

Reply:
Oh my bad, I can't do this, I didn't notice the `&` in the end.